2022-03-31 18:40:27,722 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-03-31 18:40:27,722 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\python3.9.6\lib\site-packages\instabot\bot\bot.py
2022-03-31 18:40:27,725 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-03-31 18:40:29,743 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-03-31 18:40:30,312 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-03-31 18:40:30,835 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-03-31 18:40:31,207 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-03-31 18:40:32,682 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-03-31 18:40:32,696 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXEb79ghTRg4iW4zKoxq3mnunpPmGl_PhXPodnZfOAspqrx7W0fUeFMXiVbZqa4bSit2KwI/jIW2nouTS1XNbr0E/","api_path":"/challenge/AXEb79ghTRg4iW4zKoxq3mnunpPmGl_PhXPodnZfOAspqrx7W0fUeFMXiVbZqa4bSit2KwI/jIW2nouTS1XNbr0E/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-03-31 18:40:32,696 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-03-31 18:40:32,696 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-03-31 18:40:32,696 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
2022-03-31 18:40:32,696 - instabot version: 0.117.0 (api) - CRITICAL - Not logged in!
